packages/spotlight/vite.config.ts

```diff
@@ -14,12 +15,13 @@ export default defineConfig({
       entry: {
         overlay: resolve(__dirname, 'src/overlay.ts'),
         sidecar: resolve(__dirname, 'src/sidecar.ts'),
+        'vite-plugin': resolve(__dirname, 'src/vite-plugin.ts'),
       },
       // the proper extensions will be added
       //   fileName: 'sentry-spotlight',
     },
     rollupOptions: {
-      external: dependencies,
+      external: [...dependencies, 'node:path', 'node:crypto'],
```

Comment:
l: same thought around `builtinModules` here

Reply:
Same as above, will definitely give it a look but not in this PR.

Reply:
#441